src/pages/profession/forms/ProfessionsForm.js

```diff
@@ -16,203 +16,193 @@ import CustomTextField from 'src/components/Inputs/CustomTextField'
 import { RemittanceSettingsRepository } from 'src/repositories/RemittanceRepository'
 import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
 import { DataSets } from 'src/resources/DataSets'
+import CustomNumberField from 'src/components/Inputs/CustomNumberField'
 
-export default function ProfessionsForm ({labels, maxAccess,recordId}){
+export default function ProfessionsForm({ labels, maxAccess, recordId }) {
   const [isLoading, setIsLoading] = useState(false)
   const [editMode, setEditMode] = useState(!!recordId)
   const [position, setPosition] = useState(0)
-  
+
   const [initialValues, setInitialData] = useState({
-      recordId: null,
-      reference: '',
-      name: '',
-      flName: '',
-      monthlyIncome: '',
-      riskFactor: '',
-      diplomatStatus:''
-    })
-
-    const { getRequest, postRequest } = useContext(RequestsContext)
-
-    const invalidate = useInvalidate({
-      endpointId: RemittanceSettingsRepository.Profession.page,
-    })
-
-    const formik = useFormik({
-      initialValues,
-      enableReinitialize: true,
-      validateOnChange: true,
-      validationSchema: yup.object({
-        name: yup.string().required(' '),
-        reference: yup.string().required(' '),
-        flName: yup.string().required(' '),
-        monthlyIncome: yup.string().required(' '),
-        riskFactor: yup.string().required(' '),
-        diplomatStatus: yup.string().required(' '),
-      }),
-      onSubmit: async obj => {
-        try{
+    recordId: null,
+    reference: '',
+    name: '',
+    flName: '',
+    monthlyIncome: '',
+    riskFactor: '',
+    diplomatStatus: ''
+  })
+
+  const { getRequest, postRequest } = useContext(RequestsContext)
+
+  const invalidate = useInvalidate({
+    endpointId: RemittanceSettingsRepository.Profession.page
+  })
+
+  const formik = useFormik({
+    initialValues,
+    enableReinitialize: true,
+    validateOnChange: true,
+    validationSchema: yup.object().shape({
+      name: yup.string().required('Name is required'),
```

Comment:
remove the message when the field is required. we only need the field to become red
name: yup.string().required(' '),
do the same for the rest